synapse/rest/client/v1/admin.py

```diff
@@ -66,6 +69,25 @@ def on_GET(self, request, user_id):
         defer.returnValue((200, ret))
 
 
+class VersionServlet(ClientV1RestServlet):
+    PATTERNS = client_path_patterns("/admin/server_version")
```

Comment:
I guess a first point to make sure we agree on is what to call this endpoint.

`server_version` seems better  than simply `version` IMO, because it is more explicit

Reply:
Also I just realised I forgot to add documentation for this endpoint.